modules/imap/site.js

```diff
@@ -822,32 +869,32 @@ var get_local_message_content = function(msg_uid, path) {
     return Hm_Utils.get_from_local_storage(getMessageStorageKey(msg_uid));
 };
 
-var imap_setup_message_view_page = function(uid, details, list_path, callback) {
+var imap_setup_message_view_page = function (uid, details, list_path, callback) {
     if (!uid) {
         uid = getMessageUidParam();
     }
-    const callbackFn = (...args) => {        
```

Comment:
Could you avoid such changes in the future? it's really difficult to focus on what has actually changed.